plugins/legacy-preset-chart-big-number/src/BigNumberTotal/controlPanel.ts

```diff
@@ -43,6 +48,36 @@ export default {
           },
         ],
         ['y_axis_format'],
+        [
+          {
+            name: 'header_format_selector',
+            config: {
+              type: 'CheckboxControl',
+              label: t('Timestamp Format'),
+              renderTrigger: true,
+              default: false,
+              description: t('Whether to format the timestamp'),
+            },
+          },
+        ],
+        [
+          {
+            name: 'header_timestamp_format',
+            config: {
+              type: 'SelectControl',
+              freeForm: true,
+              label: t('Timestamp format'),
```

Comment:
Can we name it `Date format` for consistency with other charts?
Also, instead of adding a checkbox  to control visibility, why don't we just make this select button clearable?

Reply:
We need an explicit way to tell the user whether to use `number formatter` or `time formatter`